libs/traits/src/lib.rs

```diff
@@ -435,6 +435,13 @@ pub mod fees {
 			fee: Fee<Self::Balance, Self::FeeKey>,
 		) -> DispatchResult;
 	}
+
+	/// Trait to pay fees
+	/// This trait can be used by pallet to just pay fees without worring about
+	/// the value or where the fee goes.
+	pub trait PayFee<AccountId> {
+		fn pay(who: &AccountId) -> DispatchResult;
```

Comment:
Given the previous discussion, I would for sure document that `who` is the account that's paying for the fees and not the account that is receiving them.

Reply:
+1 on that. Would also be great to rename `who` to `payer`.

Reply:
Good point! Sorry for the "auto-merge", keep in mind for a future PR where I could add this 🧠

Reply:
Here it goes: https://github.com/centrifuge/centrifuge-chain/pull/1629/commits/2901f0ec4d758715b5bbdc89feba843590496944